include/umf/memory_pool_ops.h

```diff
@@ -17,6 +17,25 @@
 extern "C" {
 #endif
 
+/// @brief This structure comprises function pointers used by corresponding umfMemoryPool calls.
+typedef struct umf_memory_pool_ext_ops_t {
+    ///
+    /// @brief Control operation for the memory pool.
+    ///        The function is used to perform various control operations
+    ///        on the memory pool.
+    ///
+    /// @param hPool handle to the memory pool.
+    /// @param operationType type of the operation to be performed.
+    /// @param name name associated with the operation.
+    /// @param arg argument for the operation.
+    /// @param query_type type of the query to be performed.
+    ///
+    /// @return umf_result_t result of the control operation.
+    ///
+    umf_result_t (*ctl)(void *hPool, int operationType, const char *name,
+                        void *arg, umf_ctl_query_type_t query_type);
```

Comment:
please not mix the styles

query_type -> queryType

Reply:
Done